tinygrad/tensor.py

```diff
@@ -93,20 +90,20 @@ def shape(self):
     return self.data.shape
 
   @staticmethod
-  def zeros(*shape):
-    return Tensor(np.zeros(shape, dtype=np.float32))
+  def zeros(shape, gpu=False):
```

Comment:
Why remove the stars?

Reply:
zeros and ones have two interfaces in pytorch: () and a, b, c. I think tuple is more commenly used, therefore this change. randn is different on a, b, c interface in pytorch.

Reply:
this is wrong, pytorch.randn has a tuple and a list interface, but numpy.random.randn has only a list interface...